meta-cli/src/typegraph/postprocess.rs

```diff
@@ -111,7 +155,9 @@ pub mod deno_rt {
                 let mut mat_data: ModuleMatData = object_from_map(std::mem::take(&mut mat.data))?;
                 if !mat_data.code.starts_with("file:") {
                     // TODO check imported functions exist
-                    mat_data.code = transform_module(mat_data.code)?;
+                    let path = mat_data.code.strip_prefix("file:").unwrap();
```

Comment:
Won't this always fail if ever executed??

Reply:
I do not understand, the if statement above guarantees the prefix is still present so unwrap should never fail.

Reply:
The condition is if not starting with `file:`...